operator/internal/pkg/prepare/deploy.go

```diff
@@ -24,12 +25,18 @@ import (
 	"github.com/kyma-project/module-manager/operator/pkg/util"
 )
 
+type InstallCfgType string
+
 const (
-	configReadError = "reading install %s resulted in an error for " + v1alpha1.ManifestKind
+	InstallTypeSecret InstallCfgType = "local-secret"
+	InstallTypeClient InstallCfgType = "local-client"
+	configReadError                  = "reading install %s resulted in an error for " + v1alpha1.ManifestKind
 )
 
+var LocalClient func() *rest.Config //nolint:gochecknoglobals
```

Comment:
why do you need this function variable? couldnt you just create it on the fly?

Reply:
since this is set by tests and is referred in this file

Reply:
Understood. could you add a comment here explaining that?

Reply:
this implementation was changed as discussed.